test/new_user_test.rb

```diff
@@ -0,0 +1,10 @@
+require './lib/joke'
```

Comment:
I would probably just delete this file, since you have a UserTest in the user_test.rb file :)